.github/workflows/auto_tag_stable.yml

```diff
@@ -37,5 +37,11 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
+          git config --global user.email "PROJ-bot@example.com"
+          git config --global user.name "PROJ-bot"
+          touch .dummy-file
+          git add .dummy-file
+          # Do that so that stable doesn't have the same git sha as the stable branch, so ReadTheDocs triggers a build
+          git commit -a -m "Add .dummy-file"
```

Comment:
(minor comment) Do you need the `-a`? You are adding it in the line above.

Reply:
letting that for now, because it works and doesn't hurt